### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by TarakanovAndrey** [Hexlet/hexlet-friends#322]
## Ожидаемое поведение

В разделе "Нерешенные проблемы" должны выводится открытые issues.
## Текущее поведение

Выводятся и открытые и закрытые issues.

**Comment by ajib6ept** [comment]
беру в работу

**Comment by ajib6ept** [comment]
не получается решить проблему (не смог воспроизвести ошибку у себя), задача свободна

**Comment by DREU007** [comment]
@ajib6ept, смог воспроизвести с фильтрами.

1. Перешёл на /issues/
2. Language: Python
3. Status: Open

Выдаёт таблицу с закрытыми и открытыми issues.
Closed ID: 41, 53, 90 и т.д.
Open ID: 65, 100 и т.д.

![image](https://github.com/Hexlet/hexlet-friends/assets/107818799/169fd810-6eaf-4c75-ab22-ae4b164e10cf)

**Comment by ajib6ept** [comment]
Да, на проде я ошибку видел - развернул версию локально, загрузил данными и локально у меня ошибка не воспроизвелась - поэтому задачу решить не смог и освободил для желающих

**Comment by TarakanovAndrey** [comment]
[ajib6ept](https://github.com/ajib6ept) [DREU007](https://github.com/DREU007) Скорее всего, это потому что сейчас не реализован или не работает механизм обновления статуса задачи, который реализовывается через вебхуки. А когда разворачиваешь локально, то получаешь загрузку текущего состояния задач из репозиториев, поэтому и не видишь закрытых issues
